litmus-portal/frontend/src/pages/Community/styles.ts

```diff
@@ -46,17 +39,20 @@ const useStyles = makeStyles((theme) => ({
       marginLeft: theme.spacing(8),
     },
   },
+
   paper: {
     flexGrow: 1,
     marginRight: theme.spacing(2),
     textAlign: 'center',
     color: theme.palette.text.secondary,
+    backgroundColor: '#FFF',
```

Comment:
Use theme colors

Reply:
For background we don't have this #FFF as of now in the palette.